examples/token-stats/methods/guest/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "token-stats"
+version = "0.1.0"
+edition = "2021"
+
+[workspace]
+
+[dependencies]
+alloy-sol-types = { version = "0.7" }
+core = { path = "../../core" }
+risc0-steel = { path = "../../../../steel" }
+risc0-zkvm = { git = "https://github.com/risc0/risc0", branch = "main", default-features = false, features = ["std"] }
```

Comment:
It'd be better to pin to a specific commit rather than using the main branch. It's very easy for this to break in the future.